bin/deploy.sh

```diff
@@ -158,7 +165,7 @@ deploy_charts()
 isalive_check()
 {
     PROTO="http"
-    if $(grep -q 'useTLS:\s*true' ${CFGDIR}/common.yaml ${CFGDIR}/openam.yaml); then
+    if $(grep -q '^useTLS:\s*true' ${CFGDIR}/common.yaml ${CFGDIR}/openam.yaml); then
```

Comment:
perhaps we should just bite the bullet and not allow http:// at all
It would simplify deployment.
We just need a way to install self signed for minikube

Reply:
Seems like it shouldn't be that difficult to generate a self-signed cert and install it